File: platform/src/main/java/ru/abondin/hreasy/platform/HreasyPlatformApplication.java
package ru.abondin.hreasy.platform;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.security.oauth2.server.servlet.OAuth2AuthorizationServerJwtAutoConfiguration;
import org.springframework.boot.autoconfigure.thymeleaf.ThymeleafAutoConfiguration;

@SpringBootApplication
@EnableAutoConfiguration(exclude = {
        ThymeleafAutoConfiguration.class,
        OAuth2AuthorizationServerJwtAutoConfiguration.class
})
public class HreasyPlatformApplication {

    public static void main(String[] args) {
        SpringApplication.run(HreasyPlatformApplication.class, args);
    }

}

File: platform/src/main/kotlin/ru/abondin/hreasy/platform/HreasyPlatformApplication.kt
package ru.abondin.hreasy.platform

import org.slf4j.Logger
import org.slf4j.LoggerFactory
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.security.config.annotation.web.reactive.EnableWebFluxSecurity
import org.springframework.web.reactive.config.EnableWebFlux

@SpringBootApplication
@EnableWebFluxSecurity
@EnableWebFlux
class HreasyPlatformApplication

fun main(args: Array<String>) {
    runApplication<HreasyPlatformApplication>(*args)
}

inline fun <reified T> T.logger(): Logger {
    return LoggerFactory.getLogger(T::class.java)
}
